README.adoc

```diff
@@ -425,7 +425,8 @@ inventory   inventory-guide.apps.lights.os.fyre.ibm.com         inventory   9085
 Visit the `inventory` application by going to the following URL. Make sure to substitute the appropriate `HOST`:
 `http://[HOST]/inventory/systems`
 
-For example: `\http://inventory-guide.apps.lights.os.fyre.ibm.com/inventory/systems`.
+For example, `inventory-guide.apps.lights.os.fyre.ibm.com` is the HOST in the
+`\http://inventory-guide.apps.lights.os.fyre.ibm.com/inventory/systems` URL.
```

Comment:
@MaiHameed Should "HOST" be in backticks like it is in the previous sentence?

Reply:
Good catch, fixed, thanks :)